ocdskit/cli/commands/compile.py

```diff
@@ -1,10 +1,24 @@
 from collections import defaultdict, OrderedDict
+from functools import lru_cache
 
 import ocdsmerge
 
 from .base import BaseCommand
 
 
+@lru_cache()
```

Comment:
I don't think this works, because dict's are unhashable.

Reply:
Assuming we want to allow users to pass a dict as the schema (allowed by OCDS Merge 0.5).